src/github.com/stellar/gateway/protocols/compliance/send.go

```diff
@@ -74,11 +74,13 @@ func (request *SendRequest) Validate() error {
 		return protocols.NewInvalidParameterError("destination", request.Destination, "Not a valid stellar address.")
 	}
 
-	_, err = keypair.Parse(request.AssetIssuer)
-	if !protocols.IsValidAccountID(request.AssetIssuer) {
-		return protocols.NewInvalidParameterError("asset_issuer", request.AssetIssuer, "Asset issuer must be a public key (starting with `G`).")
-	}
+	if request.AssetCode != "" {
 
+		_, err = keypair.Parse(request.AssetIssuer)
```

Comment:
This wasn't introduce in your PR but I think we can remove this line. `err` is not used anywhere and `request.AssetIssuer` is checked in the next line.

Reply:
yep, true.. removed it.